app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
+
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError
+        if (
+                "wearing_a_mask" not in visitor
+                or visitor["wearing_a_mask"] is False
+        ):
```

Comment:
You can simplify this code
```suggestion
        if not visitor.get("wearing_a_mask"):
```